script/build.ts

```diff
@@ -123,10 +123,18 @@ function packageApp(
     )
   }
 
+  const toPackageArch = (targetArch: string | undefined): packager.arch => {
+    if (targetArch === 'arm64' || targetArch === 'x64') {
```

Comment:
The `|| targetArch === 'x64'` is unnecessary here - you're returning `x64` as the fallback anyway

Reply:
@saul fair point, but I like being explicit here to indicate what is supported from user input.

@iAmWillShepherd any feels either way on this?

Reply:
If we're being explicit, shouldn't we throw if we're given an arch that we don't support?